pkg/test/resourcefixture/testdata/basic/workstations/workstation/workstation-minimal/create.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: workstations.cnrm.cloud.google.com/v1alpha1
+kind: Workstation
+metadata:
+  name: workstation-${uniqueId}
+spec:
+  parentRef:
+    name: workstationconfig-${uniqueId}
+  displayName: workstation-${uniqueId}
```

Comment:
Is displayName required?

Reply:
Fixed, displayName is not required. Removed from minimal test.